tests/fixtures/yaml_dicts/_layouts/index.liquid

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>My blog - {{ title }}</title>
+</head>
+<body>
+{{ content }}</body>
+</html>
```

Comment:
Indentation is kind of messy when doing these kinds of tests. It took like half an hour just to get all the spaces to match.